apps/omg_child_chain_rpc/config/config.exs

```diff
@@ -25,12 +28,9 @@ config :phoenix,
 config :omg_child_chain_rpc, OMG.ChildChainRPC.Tracer,
   service: :web,
   adapter: SpandexDatadog.Adapter,
-  disabled?: {:system, "DD_DISABLED", false, {String, :to_existing_atom}},
-  env: {:system, "APP_ENV"},
+  disabled?: true,
```

Comment:
please, double check this setting in the whole umbrella, I'm not sure I can follow why it's `true` and why `false` in a couple of places.

Is the rule here, that it depends on the `MIX_ENV`?
 - `test`, `dev` -> `disabled?: true`
 - the rest -> `disabled?: false`

? If so, I think here at least, this is not the case

Reply:
everywhere should be disabled by default, and you can override the setting via `DD_DISABLED`. fixing...